pkg/network/cache/infocache_test.go

```diff
@@ -28,41 +30,46 @@ var _ = Describe("Infocache", func() {
 
 	AfterEach(func() {
 		os.RemoveAll(tmpDir)
+		cacheCreator.New("").Delete()
```

Comment:
I understand you point, but then it will leave the temporary folder pilling up (it will delete only the folders/files under it).
Any suggestion on how make this both clear and still delete the whole (random generated) temp folder?